devices/tuya.js

```diff
@@ -2274,6 +2377,97 @@ module.exports = [
             ],
         },
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0601', manufacturerName: '_TZE200_p6vz3wzt'},
+        ],
+        model: 'TS0601_cover_4',
+        vendor: 'TuYa',
+        description: 'Curtain / Blind switch',
+        fromZigbee: [fz.tuya_cover, fz.ignore_basic_report, fz.cover_position_tilt, tuya.fz.datapoints],
+        toZigbee: [tz.tuya_cover_control, tz.cover_position_tilt, tuya.tz.datapoints],
```

Comment:
Do not mix up the new (`tuya.tz.datapoints`) and old (`tz.tuya_cover_control`) api. So please map all the datapoints in `tuyaDatapoints` and remove `tz.tuya_cover_control` from this list. Same goes for `fz.tuya_cover`

Reply:
thanks, much more convenient :)
will remove this PR and post a new clean

Reply:
Thanks, let me know if I can help.

Reply:
Opened new one, and actually there is something that you might know how to handle,
Now that I'm using datapoints for covers, I don't have the tilt control as I used to have, 
Do you know how should it be done to support it?
Thanks

Reply:
Does the tilt setting go via the normal Zigbee commands and not the TuYa datapoint?

Reply:
Yes, the problem is that the device holds both cover and 2 switches, switches are accessible only from datapoint (as far as I understand)

Reply:
Then using the cover control converters is fine (`tz.cover_position_tilt`, not `tz.tuya_cover_control`!) since `tz.tuya_cover_control` can be mapped through `tuyaDatapoints`.